#! /bin/sh /usr/share/dpatch/dpatch-run
## 48-fix-doxygen-cross-links.dpatch by Marek Brudka <mbrudka@aster.pl>
##
## DP: Change doxygen output directories to fit documentation package names

@DPATCH@
diff -ru ACE_wrappers.orig/etc/ace.doxygen ACE_wrappers/etc/ace.doxygen
--- ACE_wrappers.orig/etc/ace.doxygen	2010-01-06 14:34:01.000000000 +0100
+++ ACE_wrappers/etc/ace.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -82,7 +82,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/ace
+HTML_OUTPUT           = html/libace-doc
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -146,7 +146,7 @@
 #---------------------------------------------------------------------------
 
 TAGFILES              =
-GENERATE_TAGFILE      = html/ace/ACE.tag
+GENERATE_TAGFILE      = html/libace-doc/ACE.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/etc/ace_qos.doxygen ACE_wrappers/etc/ace_qos.doxygen
--- ACE_wrappers.orig/etc/ace_qos.doxygen	2010-01-06 14:34:01.000000000 +0100
+++ ACE_wrappers/etc/ace_qos.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/ace/QoS
+HTML_OUTPUT           = html/libace-doc/QoS
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -133,8 +133,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=..
-GENERATE_TAGFILE      = html/ace/QoS/ACE_QoS.tag
+TAGFILES              = html/libace-doc/ACE.tag=..
+GENERATE_TAGFILE      = html/libace-doc/QoS/ACE_QoS.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/etc/ace_rmcast.doxygen ACE_wrappers/etc/ace_rmcast.doxygen
--- ACE_wrappers.orig/etc/ace_rmcast.doxygen	2010-01-06 14:34:01.000000000 +0100
+++ ACE_wrappers/etc/ace_rmcast.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/ace/rmcast
+HTML_OUTPUT           = html/libace-doc/rmcast
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -133,8 +133,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=..
-GENERATE_TAGFILE      = html/ace/rmcast/ACE_RMCast.tag
+TAGFILES              = html/libace-doc/ACE.tag=..
+GENERATE_TAGFILE      = html/libace-doc/rmcast/ACE_RMCast.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/etc/ace_ssl.doxygen ACE_wrappers/etc/ace_ssl.doxygen
--- ACE_wrappers.orig/etc/ace_ssl.doxygen	2010-01-06 14:34:01.000000000 +0100
+++ ACE_wrappers/etc/ace_ssl.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/ace/ssl
+HTML_OUTPUT           = html/libace-doc/ssl
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -133,8 +133,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=..
-GENERATE_TAGFILE      = html/ace/ssl/ACE_SSL.tag
+TAGFILES              = html/libace-doc/ACE.tag=..
+GENERATE_TAGFILE      = html/libace-doc/ssl/ACE_SSL.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/etc/acexml.doxygen ACE_wrappers/etc/acexml.doxygen
--- ACE_wrappers.orig/etc/acexml.doxygen	2010-01-06 14:34:01.000000000 +0100
+++ ACE_wrappers/etc/acexml.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/acexml
+HTML_OUTPUT           = html/libace-doc/acexml
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -133,8 +133,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../ace
-GENERATE_TAGFILE      = html/acexml/ACEXML.tag
+TAGFILES              = html/libace-doc/ACE.tag=../libace-doc
+GENERATE_TAGFILE      = html/libace-doc/acexml/ACEXML.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/CIAO/etc/ciao_config_handlers.doxygen ACE_wrappers/TAO/CIAO/etc/ciao_config_handlers.doxygen
--- ACE_wrappers.orig/TAO/CIAO/etc/ciao_config_handlers.doxygen	2010-02-04 11:21:23.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/etc/ciao_config_handlers.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -71,7 +71,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/ciao/tools/config_handlers
+HTML_OUTPUT           = html/libtao-doc/ciao/tools/config_handlers
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -132,18 +132,18 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../../../ace \
-                        html/tao/TAO.tag=../../.. \
-                        html/ace/ssl/ACE_SSL.tag=../../../../ace/ssl \
-                        html/tao/portableserver/TAO_PortableServer.tag=../../../portableserver \
-                        html/tao/iortable/TAO_IORTable.tag=../../../iortable \
-                        html/tao/rtcorba/TAO_RTCORBA.tag=../../../rtcorba \
-                        html/tao/security/TAO_Security.tag=../../../security \
-                        html/acexml/ACEXML.tag=../../../../acexml \
-                        html/tao/rtportableserver/TAO_RTPortableServer.tag=../../../rtportableserver \
-                        html/tao/cosnaming/TAO_CosNaming.tag=../../../cosnaming \
-                        html/tao/ifr/TAO_IFR.tag=../../../ifr
-GENERATE_TAGFILE      = html/tao/ciao/tools/Config_Handlers.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../../../libace-doc \
+                        html/libtao-doc/TAO.tag=../../.. \
+                        html/libace-doc/ssl/ACE_SSL.tag=../../../../libace-doc/ssl \
+                        html/libtao-doc/portableserver/TAO_PortableServer.tag=../../../portableserver \
+                        html/libtao-doc/iortable/TAO_IORTable.tag=../../../iortable \
+                        html/libtao-doc/rtcorba/TAO_RTCORBA.tag=../../../rtcorba \
+                        html/libtao-doc/security/TAO_Security.tag=../../../security \
+                        html/libace-doc/acexml/ACEXML.tag=../../../../libace-doc/acexml \
+                        html/libtao-doc/rtportableserver/TAO_RTPortableServer.tag=../../../rtportableserver \
+                        html/libtao-doc/cosnaming/TAO_CosNaming.tag=../../../cosnaming \
+                        html/libtao-doc/ifr/TAO_IFR.tag=../../../ifr
+GENERATE_TAGFILE      = html/libtao-doc/ciao/tools/Config_Handlers.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/CIAO/etc/ciao_DAnCE.doxygen ACE_wrappers/TAO/CIAO/etc/ciao_DAnCE.doxygen
--- ACE_wrappers.orig/TAO/CIAO/etc/ciao_DAnCE.doxygen	2010-02-04 11:21:23.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/etc/ciao_DAnCE.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -71,7 +71,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/ciao/dance
+HTML_OUTPUT           = html/libtao-doc/ciao/dance
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -132,18 +132,18 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../../ace \
-                        html/tao/TAO.tag=../.. \
-                        html/ace/ssl/ACE_SSL.tag=../../../ace/ssl \
-                        html/tao/portableserver/TAO_PortableServer.tag=../../portableserver \
-                        html/tao/iortable/TAO_IORTable.tag=../../iortable \
-                        html/tao/rtcorba/TAO_RTCORBA.tag=../../rtcorba \
-                        html/tao/security/TAO_Security.tag=../../security \
-                        html/acexml/ACEXML.tag=../../../acexml \
-                        html/tao/rtportableserver/TAO_RTPortableServer.tag=../../rtportableserver \
-                        html/tao/cosnaming/TAO_CosNaming.tag=../../cosnaming \
-                        html/tao/ifr/TAO_IFR.tag=../../ifr
-GENERATE_TAGFILE      = html/tao/ciao/DAnCE.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../../libace-doc \
+                        html/libtao-doc/TAO.tag=../.. \
+                        html/libace-doc/ssl/ACE_SSL.tag=../../../libace-doc/ssl \
+                        html/libtao-doc/portableserver/TAO_PortableServer.tag=../../portableserver \
+                        html/libtao-doc/iortable/TAO_IORTable.tag=../../iortable \
+                        html/libtao-doc/rtcorba/TAO_RTCORBA.tag=../../rtcorba \
+                        html/libtao-doc/security/TAO_Security.tag=../../security \
+                        html/libace-doc/acexml/ACEXML.tag=../../../libace-doc/acexml \
+                        html/libtao-doc/rtportableserver/TAO_RTPortableServer.tag=../../rtportableserver \
+                        html/libtao-doc/cosnaming/TAO_CosNaming.tag=../../cosnaming \
+                        html/libtao-doc/ifr/TAO_IFR.tag=../../ifr
+GENERATE_TAGFILE      = html/libtao-doc/ciao/DAnCE.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/CIAO/etc/ciao_dds4ccm.doxygen ACE_wrappers/TAO/CIAO/etc/ciao_dds4ccm.doxygen
--- ACE_wrappers.orig/TAO/CIAO/etc/ciao_dds4ccm.doxygen	2010-01-06 14:32:13.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/etc/ciao_dds4ccm.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -71,7 +71,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/ciao/dds4ccm
+HTML_OUTPUT           = html/libtao-doc/ciao/dds4ccm
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -132,18 +132,18 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../../ace \
-                        html/tao/TAO.tag=../.. \
-                        html/ace/ssl/ACE_SSL.tag=../../../ace/ssl \
-                        html/tao/portableserver/TAO_PortableServer.tag=../../portableserver \
-                        html/tao/iortable/TAO_IORTable.tag=../../iortable \
-                        html/tao/rtcorba/TAO_RTCORBA.tag=../../rtcorba \
-                        html/tao/security/TAO_Security.tag=../../security \
-                        html/acexml/ACEXML.tag=../../../acexml \
-                        html/tao/rtportableserver/TAO_RTPortableServer.tag=../../rtportableserver \
-                        html/tao/cosnaming/TAO_CosNaming.tag=../../cosnaming \
-                        html/tao/ifr/TAO_IFR.tag=../../ifr
-GENERATE_TAGFILE      = html/tao/ciao/DAnCE.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../../libace-doc \
+                        html/libtao-doc/TAO.tag=../.. \
+                        html/libace-doc/ssl/ACE_SSL.tag=../../../libace-doc/ssl \
+                        html/libtao-doc/portableserver/TAO_PortableServer.tag=../../portableserver \
+                        html/libtao-doc/iortable/TAO_IORTable.tag=../../iortable \
+                        html/libtao-doc/rtcorba/TAO_RTCORBA.tag=../../rtcorba \
+                        html/libtao-doc/security/TAO_Security.tag=../../security \
+                        html/libace-doc/acexml/ACEXML.tag=../../../libace-doc/acexml \
+                        html/libtao-doc/rtportableserver/TAO_RTPortableServer.tag=../../rtportableserver \
+                        html/libtao-doc/cosnaming/TAO_CosNaming.tag=../../cosnaming \
+                        html/libtao-doc/ifr/TAO_IFR.tag=../../ifr
+GENERATE_TAGFILE      = html/libtao-doc/ciao/DAnCE.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/CIAO/etc/ciao.doxygen ACE_wrappers/TAO/CIAO/etc/ciao.doxygen
--- ACE_wrappers.orig/TAO/CIAO/etc/ciao.doxygen	2010-02-04 11:21:23.000000000 +0100
+++ ACE_wrappers/TAO/CIAO/etc/ciao.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/ciao/ciao
+HTML_OUTPUT           = html/libtao-doc/ciao/ciao
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -133,18 +133,18 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../../ace \
-                        html/tao/TAO.tag=../.. \
-                        html/ace/ssl/ACE_SSL.tag=../../../ace/ssl \
-                        html/tao/portableserver/TAO_PortableServer.tag=../../portableserver \
-                        html/tao/iortable/TAO_IORTable.tag=../../iortable \
-                        html/tao/rtcorba/TAO_RTCORBA.tag=../../rtcorba \
-                        html/tao/security/TAO_Security.tag=../../security \
-                        html/acexml/ACEXML.tag=../../../acexml \
-                        html/tao/rtportableserver/TAO_RTPortableServer.tag=../../rtportableserver \
-                        html/tao/cosnaming/TAO_CosNaming.tag=../../cosnaming \
-                        html/tao/ifr/TAO_IFR.tag=../../ifr
-GENERATE_TAGFILE      = html/tao/ciao/ciao/CIAO.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../../libace-doc \
+                        html/libtao-doc/TAO.tag=../.. \
+                        html/libace-doc/ssl/ACE_SSL.tag=../../../libace-doc/ssl \
+                        html/libtao-doc/portableserver/TAO_PortableServer.tag=../../portableserver \
+                        html/libtao-doc/iortable/TAO_IORTable.tag=../../iortable \
+                        html/libtao-doc/rtcorba/TAO_RTCORBA.tag=../../rtcorba \
+                        html/libtao-doc/security/TAO_Security.tag=../../security \
+                        html/libace-doc/acexml/ACEXML.tag=../../../libace-doc/acexml \
+                        html/libtao-doc/rtportableserver/TAO_RTPortableServer.tag=../../rtportableserver \
+                        html/libtao-doc/cosnaming/TAO_CosNaming.tag=../../cosnaming \
+                        html/libtao-doc/ifr/TAO_IFR.tag=../../ifr
+GENERATE_TAGFILE      = html/libtao-doc/ciao/ciao/CIAO.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_anytypecode.doxygen ACE_wrappers/TAO/etc/tao_anytypecode.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_anytypecode.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_anytypecode.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/anytypecode
+HTML_OUTPUT           = html/libtao-doc/anytypecode
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/anytypecode/TAO_AnyTypeCode.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/anytypecode/TAO_AnyTypeCode.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_av.doxygen ACE_wrappers/TAO/etc/tao_av.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_av.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_av.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -75,7 +75,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/av
+HTML_OUTPUT           = html/libtao-doc/av
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -138,10 +138,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=.. \
-			html/tao/portableserver/TAO_PortableServer.tag=../portableserver
-GENERATE_TAGFILE      = html/tao/av/TAO_AV.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=.. \
+			html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver
+GENERATE_TAGFILE      = html/libtao-doc/av/TAO_AV.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_compression.doxygen ACE_wrappers/TAO/etc/tao_compression.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_compression.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_compression.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/compression
+HTML_OUTPUT           = html/libtao-doc/compression
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/compression/TAO_Compression.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/compression/TAO_Compression.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_cosevent.doxygen ACE_wrappers/TAO/etc/tao_cosevent.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_cosevent.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_cosevent.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -75,7 +75,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/cosevent
+HTML_OUTPUT           = html/libtao-doc/cosevent
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -138,11 +138,11 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-	html/tao/TAO.tag=.. \
-	html/tao/portableserver/TAO_PortableServer.tag=../portableserver \
-	html/tao/esf/TAO_ESF.tag=../esf
-GENERATE_TAGFILE      = html/tao/cosevent/TAO_CosEvent.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+	html/libtao-doc/TAO.tag=.. \
+	html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver \
+	html/libtao-doc/esf/TAO_ESF.tag=../esf
+GENERATE_TAGFILE      = html/libtao-doc/cosevent/TAO_CosEvent.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_cosnaming.doxygen ACE_wrappers/TAO/etc/tao_cosnaming.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_cosnaming.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_cosnaming.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -73,7 +73,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/cosnaming
+HTML_OUTPUT           = html/libtao-doc/cosnaming
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -136,9 +136,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-        html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/cosnaming/TAO_CosNaming.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+        html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/cosnaming/TAO_CosNaming.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_cosnotification.doxygen ACE_wrappers/TAO/etc/tao_cosnotification.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_cosnotification.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_cosnotification.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -80,7 +80,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/cosnotification
+HTML_OUTPUT           = html/libtao-doc/cosnotification
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -143,12 +143,12 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-	html/tao/TAO.tag=.. \
-	html/tao/portableserver/TAO_PortableServer.tag=../portableserver \
-	html/tao/cosevent/TAO_CosEvent.tag=../cosevent \
-	html/tao/esf/TAO_ESF.tag=../esf
-GENERATE_TAGFILE      = html/tao/cosnotification/TAO_CosNotification.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+	html/libtao-doc/TAO.tag=.. \
+	html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver \
+	html/libtao-doc/cosevent/TAO_CosEvent.tag=../cosevent \
+	html/libtao-doc/esf/TAO_ESF.tag=../esf
+GENERATE_TAGFILE      = html/libtao-doc/cosnotification/TAO_CosNotification.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_costime.doxygen ACE_wrappers/TAO/etc/tao_costime.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_costime.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_costime.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -73,7 +73,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/costime
+HTML_OUTPUT           = html/libtao-doc/costime
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -136,10 +136,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-                        html/tao/TAO.tag=.. \
-                        html/tao/portableserver/TAO_PortableServer.tag=../portableserver
-GENERATE_TAGFILE      = html/tao/costime/TAO_CosTime.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+                        html/libtao-doc/TAO.tag=.. \
+                        html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver
+GENERATE_TAGFILE      = html/libtao-doc/costime/TAO_CosTime.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_costrader.doxygen ACE_wrappers/TAO/etc/tao_costrader.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_costrader.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_costrader.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -76,7 +76,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/costrader
+HTML_OUTPUT           = html/libtao-doc/costrader
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -139,10 +139,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-	html/tao/TAO.tag=.. \
-	html/tao/portableserver/TAO_PortableServer.tag=../portableserver
-GENERATE_TAGFILE      = html/tao/costrader/TAO_CosTrader.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+	html/libtao-doc/TAO.tag=.. \
+	html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver
+GENERATE_TAGFILE      = html/libtao-doc/costrader/TAO_CosTrader.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao.doxygen ACE_wrappers/TAO/etc/tao.doxygen
--- ACE_wrappers.orig/TAO/etc/tao.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao
+HTML_OUTPUT           = html/libtao-doc
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,8 +135,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../ace
-GENERATE_TAGFILE      = html/tao/TAO.tag
+TAGFILES              = html/libace-doc/ACE.tag=../libace-doc
+GENERATE_TAGFILE      = html/libtao-doc/TAO.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_dynamicany.doxygen ACE_wrappers/TAO/etc/tao_dynamicany.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_dynamicany.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_dynamicany.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -73,7 +73,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/dynamicany
+HTML_OUTPUT           = html/libtao-doc/dynamicany
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -136,8 +136,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/dynamicany/TAO_DynamicAny.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/dynamicany/TAO_DynamicAny.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_dynamicinterface.doxygen ACE_wrappers/TAO/etc/tao_dynamicinterface.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_dynamicinterface.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_dynamicinterface.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/dynamicinterface
+HTML_OUTPUT           = html/libtao-doc/dynamicinterface
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,8 +135,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/dynamicinterface/TAO_DynamicInterface.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/dynamicinterface/TAO_DynamicInterface.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_esf.doxygen ACE_wrappers/TAO/etc/tao_esf.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_esf.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_esf.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/esf
+HTML_OUTPUT           = html/libtao-doc/esf
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,10 +135,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=.. \
-			html/tao/portableserver/TAO_PortableServer.tag=../portableserver
-GENERATE_TAGFILE      = html/tao/esf/TAO_ESF.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=.. \
+			html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver
+GENERATE_TAGFILE      = html/libtao-doc/esf/TAO_ESF.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_ifr.doxygen ACE_wrappers/TAO/etc/tao_ifr.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_ifr.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_ifr.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/ifr
+HTML_OUTPUT           = html/libtao-doc/ifr
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,12 +135,12 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=.. \
-			html/tao/portableserver/TAO_PortableServer.tag=../portableserver \
-			html/tao/iortable/TAO_IORTable.tag=../iortable \
-			html/tao/cosnaming/TAO_CosNaming.tag=../cosnaming
-GENERATE_TAGFILE      = html/tao/ifr/TAO_IFR.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=.. \
+			html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver \
+			html/libtao-doc/iortable/TAO_IORTable.tag=../iortable \
+			html/libtao-doc/cosnaming/TAO_CosNaming.tag=../cosnaming
+GENERATE_TAGFILE      = html/libtao-doc/ifr/TAO_IFR.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_implrepo.doxygen ACE_wrappers/TAO/etc/tao_implrepo.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_implrepo.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_implrepo.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/implrepo_service
+HTML_OUTPUT           = html/libtao-doc/implrepo_service
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,11 +135,11 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-	html/tao/TAO.tag=.. \
-	html/tao/portableserver/TAO_PortableServer.tag=../portableserver \
-        html/tao/iortable/TAO_IORTable.tag=../iortable
-GENERATE_TAGFILE      = html/tao/implrepo_service/TAO_ImplRepo.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+	html/libtao-doc/TAO.tag=.. \
+	html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver \
+        html/libtao-doc/iortable/TAO_IORTable.tag=../iortable
+GENERATE_TAGFILE      = html/libtao-doc/implrepo_service/TAO_ImplRepo.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_iormanip.doxygen ACE_wrappers/TAO/etc/tao_iormanip.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_iormanip.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_iormanip.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/iormanipulation
+HTML_OUTPUT           = html/libtao-doc/iormanipulation
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/iormanipulation/TAO_IORManipulation.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/iormanipulation/TAO_IORManipulation.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_iortable.doxygen ACE_wrappers/TAO/etc/tao_iortable.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_iortable.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_iortable.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/iortable
+HTML_OUTPUT           = html/libtao-doc/iortable
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/iortable/TAO_IORTable.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/iortable/TAO_IORTable.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_pi.doxygen ACE_wrappers/TAO/etc/tao_pi.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_pi.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_pi.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/pi
+HTML_OUTPUT           = html/libtao-doc/pi
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/pi/TAO_PI.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/pi/TAO_PI.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_pi_server.doxygen ACE_wrappers/TAO/etc/tao_pi_server.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_pi_server.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_pi_server.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/pi_server
+HTML_OUTPUT           = html/libtao-doc/pi_server
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/pi_server/TAO_PI_Server.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/pi_server/TAO_PI_Server.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_portablegroup.doxygen ACE_wrappers/TAO/etc/tao_portablegroup.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_portablegroup.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_portablegroup.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -75,7 +75,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/portablegroup
+HTML_OUTPUT           = html/libtao-doc/portablegroup
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -138,10 +138,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-	html/tao/TAO.tag=.. \
-	html/tao/portableserver/TAO_PortableServer.tag=../portableserver
-GENERATE_TAGFILE      = html/tao/portablegroup/TAO_PortableGroup.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+	html/libtao-doc/TAO.tag=.. \
+	html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver
+GENERATE_TAGFILE      = html/libtao-doc/portablegroup/TAO_PortableGroup.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_portableserver.doxygen ACE_wrappers/TAO/etc/tao_portableserver.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_portableserver.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_portableserver.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/portableserver
+HTML_OUTPUT           = html/libtao-doc/portableserver
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/portableserver/TAO_PortableServer.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/portableserver/TAO_PortableServer.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_pss.doxygen ACE_wrappers/TAO/etc/tao_pss.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_pss.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_pss.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/pss
+HTML_OUTPUT           = html/libtao-doc/pss
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,10 +135,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=.. \
-			html/tao/portableserver/TAO_PortableServer.tag=../portableserver
-GENERATE_TAGFILE      = html/tao/pss/TAO_PSS.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=.. \
+			html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver
+GENERATE_TAGFILE      = html/libtao-doc/pss/TAO_PSS.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_rtcorba.doxygen ACE_wrappers/TAO/etc/tao_rtcorba.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_rtcorba.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_rtcorba.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/rtcorba
+HTML_OUTPUT           = html/libtao-doc/rtcorba
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/rtcorba/TAO_RTCORBA.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/rtcorba/TAO_RTCORBA.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_rtevent.doxygen ACE_wrappers/TAO/etc/tao_rtevent.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_rtevent.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_rtevent.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -81,7 +81,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/rtevent
+HTML_OUTPUT           = html/libtao-doc/rtevent
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -144,11 +144,11 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-	html/tao/TAO.tag=.. \
-	html/tao/portableserver/TAO_PortableServer.tag=../portableserver \
-	html/tao/esf/TAO_ESF.tag=../esf
-GENERATE_TAGFILE      = html/tao/rtevent/TAO_RTEvent.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+	html/libtao-doc/TAO.tag=.. \
+	html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver \
+	html/libtao-doc/esf/TAO_ESF.tag=../esf
+GENERATE_TAGFILE      = html/libtao-doc/rtevent/TAO_RTEvent.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_rtportableserver.doxygen ACE_wrappers/TAO/etc/tao_rtportableserver.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_rtportableserver.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_rtportableserver.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/rtportableserver
+HTML_OUTPUT           = html/libtao-doc/rtportableserver
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,9 +135,9 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/rtportableserver/TAO_RTPortableServer.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/rtportableserver/TAO_RTPortableServer.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_security.doxygen ACE_wrappers/TAO/etc/tao_security.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_security.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_security.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -77,7 +77,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/security
+HTML_OUTPUT           = html/libtao-doc/security
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -140,10 +140,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=.. \
-			html/tao/portableserver/TAO_PortableServer.tag=../portableserver
-GENERATE_TAGFILE      = html/tao/security/TAO_Security.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=.. \
+			html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver
+GENERATE_TAGFILE      = html/libtao-doc/security/TAO_Security.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_smartproxies.doxygen ACE_wrappers/TAO/etc/tao_smartproxies.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_smartproxies.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_smartproxies.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/smartproxies
+HTML_OUTPUT           = html/libtao-doc/smartproxies
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,8 +135,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/smartproxies/TAO_SmartProxies.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/smartproxies/TAO_SmartProxies.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_ssliop.doxygen ACE_wrappers/TAO/etc/tao_ssliop.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_ssliop.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_ssliop.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -77,7 +77,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/ssliop
+HTML_OUTPUT           = html/libtao-doc/ssliop
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -140,11 +140,11 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-			html/tao/TAO.tag=.. \
-			html/tao/portableserver/TAO_PortableServer.tag=../portableserver \
-			html/tao/security/TAO_Security.tag=../security
-GENERATE_TAGFILE      = html/tao/ssliop/TAO_SSLIOP.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+			html/libtao-doc/TAO.tag=.. \
+			html/libtao-doc/portableserver/TAO_PortableServer.tag=../portableserver \
+			html/libtao-doc/security/TAO_Security.tag=../security
+GENERATE_TAGFILE      = html/libtao-doc/ssliop/TAO_SSLIOP.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_strategies.doxygen ACE_wrappers/TAO/etc/tao_strategies.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_strategies.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_strategies.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/strategies
+HTML_OUTPUT           = html/libtao-doc/strategies
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,8 +135,8 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace html/tao/TAO.tag=..
-GENERATE_TAGFILE      = html/tao/strategies/TAO_Strategies.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc html/libtao-doc/TAO.tag=..
+GENERATE_TAGFILE      = html/libtao-doc/strategies/TAO_Strategies.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_transportcurrent.doxygen ACE_wrappers/TAO/etc/tao_transportcurrent.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_transportcurrent.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_transportcurrent.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/transportcurrent
+HTML_OUTPUT           = html/libtao-doc/transportcurrent
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,10 +135,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-                        html/tao/TAO.tag=.. \
-                        html/tao/pi/TAO_PI.tag=../pi
-GENERATE_TAGFILE      = html/tao/transportcurrent/TAO_TransportCurrent.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+                        html/libtao-doc/TAO.tag=.. \
+                        html/libtao-doc/pi/TAO_PI.tag=../pi
+GENERATE_TAGFILE      = html/libtao-doc/transportcurrent/TAO_TransportCurrent.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
diff -ru ACE_wrappers.orig/TAO/etc/tao_ziop.doxygen ACE_wrappers/TAO/etc/tao_ziop.doxygen
--- ACE_wrappers.orig/TAO/etc/tao_ziop.doxygen	2010-01-06 14:33:41.000000000 +0100
+++ ACE_wrappers/TAO/etc/tao_ziop.doxygen	2010-04-01 22:11:55.000000000 +0200
@@ -72,7 +72,7 @@
 #---------------------------------------------------------------------------
 
 GENERATE_HTML         = YES
-HTML_OUTPUT           = html/tao/ziop
+HTML_OUTPUT           = html/libtao-doc/ziop
 HTML_HEADER           =
 HTML_FOOTER           =
 HTML_STYLESHEET       =
@@ -135,10 +135,10 @@
 # Configuration::additions related to external references
 #---------------------------------------------------------------------------
 
-TAGFILES              = html/ace/ACE.tag=../../ace \
-                        html/tao/TAO.tag=.. \
-                        html/tao/compression/TAO_Compression.tag=../compression
-GENERATE_TAGFILE      = html/tao/ziop/TAO_ZIOP.tag
+TAGFILES              = html/libace-doc/ACE.tag=../../libace-doc \
+                        html/libtao-doc/TAO.tag=.. \
+                        html/libtao-doc/compression/TAO_Compression.tag=../compression
+GENERATE_TAGFILE      = html/libtao-doc/ziop/TAO_ZIOP.tag
 ALLEXTERNALS          = NO
 PERL_PATH             = /usr/bin/perl
 
